tangelo/linq/translator/__init__.py

```diff
@@ -15,7 +15,7 @@
 from tangelo.helpers.utils import installed_backends
 
 from .translate_braket import translate_braket, get_braket_gates
-from .translate_qiskit import translate_qiskit, get_qiskit_gates
+from .translate_qiskit import translate_qiskit, get_qiskit_gates, translate_c_from_qiskit, translate_c_to_qiskit
```

Comment:
Good for now !

Later on, we will stop importing those functions and instead make the user go through the "umbrella" function `translate_circuit` that takes source and target as argument, and calls the functions you have defined underneath. This way all translation functions are hidden under a single function that handles all cases.